src/Model/Group.php

```diff
@@ -556,7 +556,7 @@ public static function getIdForForum(int $id)
 	 *
 	 * @param integer $id Contact ID
 	 */
-	public static function getMembersForForum(int $id)
+	public static function UpdateMembersForForum(int $id)
```

Comment:
Code standards: Almost!
```suggestion
	public static function updateMembersForForum(int $id)
```